index.html

```diff
@@ -5,27 +5,29 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <script src="main.jss" defer></script>
+    <script src="main.js" defer></script>
     <title>TodoList</title>
 </head>
 <body>
     <div id="flex-container">
 
         <div id="header">
-            <h1>Todo List</h1>
+            <h1>Todo-List</h1>
         </div>
 
         <div id="add-todo">
             <label id="add-label"><Strong>Add Item</Strong></label><br><br>
             <input type="text" placeholder="Add todo worklist" id="input-item">
-            <span><button id="add-button">Submit</button></span>
+            <span>
+                  <button id="button" class="submit">Add-Todo</button>
+                  <button id="button" class="reset" onclick="Reset()">Reset</button>
```

Comment:
use camelCase for function names, so ```Reset``` and ```Clear``` will be ```reset``` and ```clear```

Reply:
Done sir